gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/handler/GremlinResponseHandler.java

```diff
@@ -44,6 +46,7 @@
 public class GremlinResponseHandler extends SimpleChannelInboundHandler<ResponseMessage> {
     private static final Logger logger = LoggerFactory.getLogger(GremlinResponseHandler.class);
     private static final AttributeKey<ResponseException> CAUGHT_EXCEPTION = AttributeKey.valueOf("caughtException");
+    private static final AttributeKey<Throwable> SSL_EXCEPTION = AttributeKey.valueOf("sslException");
```

Comment:
This is duplicated in two classes. Consider moving to `StateKey`.

Reply:
Removed duplication by changing the AttributeKey in the GremlinResponseHandler to public (as it is the class that adds the attribute to the context) and changing HttpGremlinRequestEncoder to reuse the AttributeKey from GremlinResponseHandler.